### DISCUSSION TITLE
Why is this train collecting iron plates even though there is no station requesting iron plates?

### DISCUSSION THREAD

**Discussion by dft95** [Opinionated-Blueprints/10-Books-Full-of-Rails#52] [Q&A] [answered]
It's backing up my rail line so the iron ore can't get dropped off. What can I do so that the iron plate provider is inactive until i later make a new stop that is an iron plate requester?
![unt1](https://github.com/user-attachments/assets/12a80016-a74f-4ac0-b0c4-61ed4c641133)

**Comment by Bocian-1** [comment, marked as answer]
You can't. This is intended and desired behaviour.
See #38

**Reply by dft95** [reply]
So I'm guessing make these two stations inline so that they don't step over each other
I'll do that, thanks
